package.json

```diff
@@ -196,6 +201,11 @@
                 }
             ],
             "view/title": [
+                {
+                    "command": "aws.submitFeedback",
+                    "when": "view == aws.explorer",
+                    "group": "navigation@5"
```

Comment:
You might want to change the @ number. There's a nonzero chance that this will swap places with the refresh button (since both are listed in position 5)

Reply:
Thanks. I couldn't find any doc for the @ stuff. So does higher number => goes to the right?

Reply:
I'm not sure how it works in this context, that would be my assumption. For the vertical menus (like the ... menu), it goes in numerical -> alphabetical order in front of the @, and numerically after the @.

Reply:
Oh, perhaps it's just lexicographically sorting them then? Wonder if blah@10 goes before blah@2

Reply:
I think the answer is yes, but I think `navigation` might be reserved in this case for the top bar. I could be wrong though. Other menus always place `navigation` at the front, no matter the alphabetization: https://code.visualstudio.com/api/references/contribution-points#contributes.menus